okapi-core/src/main/java/org/folio/okapi/util/DepResolution.java

```diff
@@ -657,8 +659,14 @@ static void installMaxIterations(
     if (!errors.isEmpty()) {
       throw new OkapiError(ErrorType.USER, String.join(". ", errors));
     }
+    Map<ModuleDescriptor, List<String>> permErrors = checkPermissionNames(modsAvailable,
+        modsEnabled);
+    for (Map.Entry<ModuleDescriptor,List<String>> ent : permErrors.entrySet()) {
+      for (String msg : ent.getValue()) {
+        logger.info("{}: {}", ent.getKey().getId(), msg);
+      }
+    }
```

Comment:
Could be extracted into a method logPermissionErrors(modsAvailable, modsEnabled).

Could be wrapped into `if (logger.isInfoEnabled() { ... }`.

Reply:
I think the info should be warn and seems overkil to check for that being enabled. This code, is only executed rarely (compared to proxy calls).